api/swagger.yaml

```diff
@@ -452,6 +463,21 @@ paths:
                   $ref: '#/definitions/ApiInfo'
         default:
           $ref: '#/responses/UnknownError'
+    post:
+      summary: 'Create API inventory item'
+      parameters:
+        - in: body
+          name: body
+          schema:
+            $ref: '#/definitions/CreateApiInfo'
```

Comment:
Why not using ApiInfo?

Reply:
because we need to provide apiType

Reply:
then I think it will be better if CreateApiInfo will contain ApiInfo and type instead of duplicating ApiInfo fields.

Reply:
And I will probably would not call it CreateApiInfo, maybe something more general like ApiInfoWithType that can be reused later?